upload/docker_images/checksummer/checksummer.py

```diff
@@ -3,14 +3,14 @@
 import argparse
 import os
 import sys
-
```

Comment:
Please follow the conventions of the codebase you're working in, or ask.  I believe the data store also uses this convention.  That's where I got it, and it makes sense.